environments/mainnet/contracts/aragonOS/0.7/KernelProxy.sol

```diff
@@ -0,0 +1,310 @@
+
+// File: contracts/acl/IACL.sol
```

Comment:
Should we add the "version tag" to these new flattened contracts?

Reply:
Forgot to add it to the contracts that have already been verified (all of the contracts in `aragonOS/0.7`) 😣, but it is possible to see which is the contract that has the same bytecode and was verified and when it was deployed.